cmd/groundstation/main.go

```diff
@@ -0,0 +1,28 @@
+package main
```

Comment:
Why are we bringing this back? It was intentionally deleted

Reply:
I think switching from an old branch caused some issues, I'll be fixing this soon. Should have labelled the PR draft :P